src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonPropertyInfo.cs

```diff
@@ -710,13 +799,26 @@ internal void EnsureChildOf(JsonTypeInfo parent)
             }
         }
 
+        internal bool TryPopulate(scoped ref ReadStack state)
```

Comment:
Seems this should be renamed to:
```suggestion
        internal bool TryGetPrePopulatedValue(scoped ref ReadStack state)
```

Reply:
will be changed in the next iteration